sql/moz-fx-data-shared-prod/monitoring_derived/event_monitoring_aggregates_v1/backfill.yaml

```diff
@@ -0,0 +1,14 @@
+2025-09-05:
+  start_date: 2025-06-01
```

Comment:
Doing 90 days because nobody has tried to query before 2025-06-06 recently based on the active/long-term storage stats